emba.sh

```diff
@@ -341,6 +341,9 @@ main()
     PRE_CHECK=0
     print_output "[*] Firmware directory detected." "no_log"
     print_output "    Emba starts with testing the environment." "no_log"
+    print_output "    The provided firmware will be copied to ""$FIRMWARE_PATH_CP" "no_log"
+    cp -R "$FIRMWARE_PATH" "$FIRMWARE_PATH_CP""/""$(basename "$FIRMWARE_PATH")"
+    FIRMWARE_PATH="$FIRMWARE_PATH_CP""/""$(basename "$FIRMWARE_PATH")"
```

Comment:
I think we are having an issue with our OUTPUT_DIR which is only feeded in module P05 and we are using it in S09. We need to set it up here to ensure it is setup correctly. Include as easy fix 
OUTPUT_DIR="$FIRMWARE_PATH_CP"